### PR TITLE
Improve ruby installation guide

### PR DESCRIPTION
**Pull request by shadre**

### CODE REVIEW

File: libraries/knapsack_pro-ruby/guide.md [hunk 1]
```diff
@@ -57,30 +55,28 @@ Please answer questions to get basic configuration of knapsack_pro gem for your
...
   <strong>What is your CI provider?</strong>
   <ul class="none-list">
-    <li><label><input type="radio" name="ci-provider" value="circleci"> https://circleci.com</label></li>
-    <li><label><input type="radio" name="ci-provider" value="travis-ci"> https://travis-ci.org</label></li>
-    <li><label><input type="radio" name="ci-provider" value="buildkite"> https://buildkite.com</label></li>
-    <li><label><input type="radio" name="ci-provider" value="semaphoreci"> https://semaphoreci.com</label></li>
-    <li><label><input type="radio" name="ci-provider" value="snap-ci"> https://snap-ci.com</label></li>
-    <li><label><input type="radio" name="ci-provider" value="codeship"> http://codeship.com</label></li>
-    <li><label><input type="radio" name="ci-provider" value="heroku-ci"> Heroku CI</label></li>
-    <li><label><input type="radio" name="ci-provider" value="solano-ci"> Solano CI</label></li>
```
Comment (shadre) [inline libraries/knapsack_pro-ruby/guide.md:67]: Solano CI is dead

File: libraries/knapsack_pro-ruby/guide.md [hunk 2]
```diff
@@ -672,146 +612,108 @@ Note the <a href="https://devcenter.heroku.com/articles/heroku-ci-parallel-test-
...
-# Step for Spinach for second CI node
+# Step for Spinach for the second CI node
 KNAPSACK_PRO_CI_NODE_TOTAL=2 KNAPSACK_PRO_CI_NODE_INDEX=1 bundle exec rake knapsack_pro:spinach
 {% endhighlight %}
 
-<p>
-Please remember to set up API token like <i>KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC</i> as global environment.
-</p>
-</div>
+    <p>
```
[the remark this answers is not in the document]
Comment (shadre) [reply libraries/knapsack_pro-ruby/guide.md]: Fixed. Thanks for noticing

**Review by ArturT — APPROVED** [review]
Looks good. 👍  I added suggestions.

**Comment by shadre** [comment]
> Looks good. +1 I added suggestions.

Great review, thank you. :+1: I used most of your suggestions.

**Pull request merged**